src/blackgui/components/configsimulatorcomponent.cpp

```diff
@@ -72,7 +72,8 @@ namespace BlackGui::Components
         ui->cb_FG->setChecked(fg);
 
         ui->cb_P3D->setEnabled(CBuildConfig::isCompiledWithP3DSupport());
-        ui->cb_FSX->setEnabled(CBuildConfig::isCompiledWithFsxSupport());
+        // TOOO remove MSFS check once it has its own entry
+        ui->cb_FSX->setEnabled(CBuildConfig::isCompiledWithFsxSupport() || CBuildConfig::isCompiledWithMsFlightSimulatorSupport());
```

Comment:
At  https://github.com/swift-project/pilotclient/blob/2ed20070486afdc8018222731ad08da4c10fef97/src/blackgui/components/configsimulatorcomponent.cpp#L81 the MSFS check must be added as well. Otherwise the checkbox is still disabled in my tests because it is set to readonly.

Reply:
Ugh, I'm sorry about this one too. I discovered that problem while testing the 64-bit build and fixed it in an amended commit like we do where I work and force pushed it to my fork's branch but my repo inherited the ban on force pushes from your repo. 🤦‍♂️

Once I'm home again tomorrow I'll get this and the other issue taken care of.

Reply:
We allow force pushes so I'm not sure what you mean by inherited.

Reply:
@ajungren Any news on this?